### DISCUSSION TITLE
21 - Resilience

### DISCUSSION THREAD

**Discussion by giscus** [SadmanYasar/sadman-yasar-blogs#16] [Announcements]
# 21 - Resilience

Sadman Yasar Sayem personal website using Next.js

https://sadman-yasar-sayem-blogs.vercel.app/posts/21

**Comment by Daniyal9988** [comment]
its heart breaking for me to hear about your fathers death. May Allah give your father highest ranks in jannah. I have been following your blogs and have been reading them its heart warming to see your success and the way you cope up after your father. I wish nothing but more success i hope you can achieve all the things you want to achieve may Allah give you more success it gives me alot of joy to go through your work and things you have worked on and the depth of your knowledge amazes me every time. Keep going brother may Allah be with you.

**Reply by SadmanYasar** [reply]
Thank you very much brother. I'm very happy to hear that you read my blogs. Since the time of this writing, I have recovered from this and now focusing on studies and some opportunities. I have plans to share more in the future blogs.
